app/api/fetch-from-upi.jsx

```
@@ -0,0 +1,65 @@
+import { PrismaClient } from "@prisma/client";
+import { extractInfo } from "../../hooks/helper";
+export const action = async ({ params, request }) => {
+  try {
+    const prisma = new PrismaClient();
+    const { text } = await request.json();
+    const sourceId = parseInt(params.sourceId, 10);
+    if (!text) {
+      return new Response("No text");
+    }
+    if (isNaN(sourceId)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)